source/user-manual/reference/ossec-conf/wodle-s3.rst

```diff
@@ -368,7 +368,7 @@ Name of AWS organization. Only works with CloudTrail buckets.
 discard_regex
 ^^^^^^^^^^^^^
 
-A regular expression to determine if an event must be discarded. It requires a mandatory ``field`` attribute. The regex is applied to the event field specified with this attribute.
+A regular expression to determine if an event must be discarded. It requires a mandatory ``field`` attribute which must be present in the fetched event (e.g., `CloudTrail log event reference <https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-event-reference.html>`_) and relative to it. The regex is applied to the event field specified with this attribute.
```

Comment:
```suggestion
A regular expression to determine if an event must be discarded. It requires a mandatory ``field`` attribute. If the field is present in the event log, the regex is applied to it.
```

Reply:
Is it possible to add an  example rather than a link and say something like the following

"... For example, `userIdentity.principalId` for the following AWS CloudTrail log example:

{
    "eventVersion": "1.09",
    "userIdentity": {
        "type": "IAMUser",
        "principalId": "EXAMPLE6E4XEGITWATV6R",
        "arn": "arn:aws:iam::123456789012:user/Mary_Major",
        ...
"

or similar?

Reply:
We could add a note explaining that it's the raw event log and not the event processed and stored in wazuh archives